### ISSUE TITLE
on pressing F1, the following message appears 

### ISSUE THREAD

**Issue by krxna** [spikespaz/search-deflector#235]
https://user-images.githubusercontent.com/66514485/142717713-4eec0ac3-4f22-4305-b45e-175c55ef001f.jpg

**Comment by LeonVQZ** [comment]
Yeah, that's probably because the microsoft-edge in the passed URI: microsoft-edge:https://go.microsoft.com/fwlink/?LinkId=528882

Maybe it can be fixed changing a registry setting, not sure tho.
